core/benches/shredder.rs

```diff
@@ -152,30 +127,50 @@ fn bench_deserialize_hdr(bencher: &mut Bencher) {
 
 #[bench]
 fn bench_shredder_coding(bencher: &mut Bencher) {
-    let symbol_count = DATA_SHREDS_PER_FEC_BLOCK;
-    let data_shreds = make_shreds(symbol_count);
+    let txs_per_entry = 128;
+    let num_entries = max_entries_per_n_shred(&make_test_entry(txs_per_entry), 200, Some(1000));
+    let entries = make_large_unchained_entries(txs_per_entry, num_entries);
```

Comment:
nit: I know `make_shreds()` needs to go away, but we could probably still have a helper `make_entries()` as this code is identical in both benches that you updated

Reply:
Done in [755f6c0](https://github.com/anza-xyz/agave/pull/5985/commits/755f6c02430d083068a202ed54a60dec11ebf1b2)